docs/terminology.md

```diff
@@ -0,0 +1,95 @@
+# Terminology
+
+## Overview
+This document contains all terminology used across compass documentation.
+
+## Management Plane 
+Abbr.: *MP*\
+\
+Includes *Compass*, *Runtime Provisioners*, *ServiceMesh* and *UI* components.
+
+The multi-tenant system which allows to:
+- create Applications
+- create Runtimes
+- manage Applications and Runtimes
+
+### MP Compass
+Abbr.: *Compass*\
+\
+Includes *Connector*, *Gateway*, *Director* and *Healtchecker* components.
```

Comment:
Maybe it should be mentioned that for now HealthCheck will be a part of Director?

Reply:
Done, I've remved healthchecker component from terminology and moved that information directly into Director.